src/javascript/app/pages/user/view_popup/view_popup.js

```diff
@@ -235,7 +235,7 @@ const ViewPopup = (() => {
         containerSetText('trade_details_ref_id', `${contract.transaction_ids.buy} (${localize('Buy')}) ${contract.transaction_ids.sell ? `<br>${contract.transaction_ids.sell} (${localize('Sell')})` : ''}`);
         containerSetText('trade_details_indicative_price', indicative_price ? formatMoney(contract.currency, indicative_price) : '-');
 
-        if (!is_ended && final_price) {
+        if (final_price) {
```

Comment:
won't this always be true? because we will always have indicative price, so original issue doesn't get resolved